vcpkg.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
+  "name": "parquetsharp",
+  "version": "5.0.0",
```

Comment:
I don't think the version here really matters, as far as I can tell it's only important if we're publishing ParquetSharp itself as a vcpkg package. I guess we should either have a process to keep it up to date, or set it to something obviously wrong like `0.0.0`